docs/content/en/options.md

```diff
@@ -168,11 +168,11 @@ This also helps making consistent requests in both SSR and Client Side code.
 
 ## `proxyHeadersIgnore`
 
-* Default `['accept', 'host', 'x-forwarded-host', 'cf-ray', 'cf-connecting-ip', 'content-length', 'content-md5', 'content-type']`
+* Default `['accept', 'host', 'x-forwarded-host', 'x-forwarded-port', 'x-forwarded-proto', 'cf-ray', 'cf-connecting-ip', 'content-length', 'content-md5', 'content-type']`
```

Comment:
Shall we also ignore `x-forwarded-for`?

Reply:
This one contains the IP of the client.
I think that's the only one that could make sense.

Reply:
Makes sense to keep